Arm-Subsytem/Hand/human_hand.hpp

```diff
@@ -119,6 +121,15 @@ class Hand : public HandInterface
       case MissionControlData::HandModes::kRoll:
         SetWristRollPosition(hand_data.wrist_data.roll, speed);
         break;
+      case MissionControlData::HandModes::kTransport:
+        thumb_.SetPosition(transport_angles);
+        pointer_.SetPosition(transport_angles);
+        middle_.SetPosition(transport_angles);
+        ring_.SetPosition(transport_angles);
+        pinky_.SetPosition(transport_angles);
+        wrist_.HandleWristMovement(transport_angles, transport_angles);
```

Comment:
We should probably make this a function to clean this up a bit. Ideally this function wouldn't take any arguments either.